cms/plugin_base.py

```diff
@@ -386,22 +391,23 @@ def get_child_plugin_candidates(cls, slot, page):
         from cms.plugin_pool import plugin_pool
         return plugin_pool.get_all_plugins()
 
-    def get_child_classes(self, slot, page):
+    @classmethod
+    def get_child_classes(cls, slot, page):
```

Comment:
We're closing the door to the possibily for the users to customize the parent/child classes based on the plugin instance. Not sure of the tradeoff